dns_check/tests/mocks.py

```diff
@@ -11,7 +11,14 @@ def __init__(self, address):
 
     class MockRrset:
         def __init__(self, address):
-            self.items = [MockDNSAnswer.MockItem(address)]
+            addresses = [x.strip().lower() for x in address.split(',')]
+            if addresses > 1:
+                items = []
+                for address in addresses:
+                    items.append(MockDNSAnswer.MockItem(address))
+                    self.items = items
```

Comment:
this should go outside the `for` body (final result is the same but you're overwriting it at every iteration)